### ISSUE TITLE
running with only `types-have-descriptions` rule still results in parsing errors

### ISSUE THREAD

**Issue by stevenqzhang** [cjoudrey/graphql-schema-linter#154]
Is it intentional that I still get parsing errors when I'm running with the `--rules types-have-descriptions` flag?

I.e. here's the error I get:

```
It would be super helpful if you could report this here: https://github.com/cjoudrey/graphql-schema-linter/issues/new

Error: Type "Person" not found in document.
    at ASTDefinitionBuilder._resolveType (/Users/steven/.config/yarn/global/node_modules/graphql-schema-linter/node_modules/graphql/utilities/buildASTSchema.js:134:11)
    at ASTDefinitionBuilder.buildType (/Users/steven/.config/yarn/global/node_modules/graphql-schema-linter/node_modules/graphql/utilities/buildASTSchema.js:218:79)
    at /Users/steven/.config/yarn/global/node_modules/graphql-schema-linter/node_modules/graphql/utilities/buildASTSchema.js:209:20

```

I would expect in this case that the linter only checks for descriptions, and does not need to know whether the schema is valid or not

**Comment by cjoudrey** [comment]
Thanks for opening this issue @stevenqzhang. 😄 

Is it possible that the `Person` type is defined in another `.graphql` file and when you can the linter you didn't provide it that file? Could you paste the full command you used to run the linter? That'd be super helpful. 😄 

On a related note, I realize this is not the greatest error, but the linter currently uses GraphQL JS' [`buildASTSchema`](https://github.com/graphql/graphql-js/blob/46b41f787159be6fa7d22ff89795f5bd3782b687/src/utilities/buildASTSchema.js) to materialize the GraphQL schema definition into a schema. Unfortunately, `buildASTSchema` is a bit unforgiving when a schema is invalid (for example, because a type is missing.) This is kinda related to #53 #70. Something I haven't gotten around fixing yet.

**Comment by cjoudrey** [comment]
Will be fixed in next release by #163.
